src/components/AssetManagementPanel/AssetManagementForm/AssetManagementForm.stories.tsx

```diff
@@ -839,6 +907,8 @@ export const EndorseTransferHolderBeneficiaryPending = () => {
       onRestoreToken={() => alert("Reject document surrender")}
       restoreTokenState={FormState.UNINITIALIZED}
       tokenId=""
+      document={obfuscatedDocument}
+      verificationStatus={whenDocumentValidAndIssuedByDns as VerificationFragment[]}
     />
   );
 };
```

Comment:
think previously already missing 1 tokenBurnt story, can add that?

Reply:
done.